cibuildwheel/windows.py

```diff
@@ -31,25 +37,28 @@ def get_nuget_args(configuration):
         python_name = python_name + "x86"
     return [python_name, "-Version", configuration.version, "-OutputDirectory", "C:/python"]
 
+
 def get_python_configurations(build_selector):
-    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier'])
+    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier', 'url'])
     python_configurations = [
-        PythonConfiguration(version='2.7.17', arch="32", identifier='cp27-win32'),
-        PythonConfiguration(version='2.7.17', arch="64", identifier='cp27-win_amd64'),
-        PythonConfiguration(version='3.5.4', arch="32", identifier='cp35-win32'),
-        PythonConfiguration(version='3.5.4', arch="64", identifier='cp35-win_amd64'),
-        PythonConfiguration(version='3.6.8', arch="32", identifier='cp36-win32'),
-        PythonConfiguration(version='3.6.8', arch="64", identifier='cp36-win_amd64'),
-        PythonConfiguration(version='3.7.5', arch="32", identifier='cp37-win32'),
-        PythonConfiguration(version='3.7.5', arch="64", identifier='cp37-win_amd64'),
-        PythonConfiguration(version='3.8.0', arch="32", identifier='cp38-win32'),
-        PythonConfiguration(version='3.8.0', arch="64", identifier='cp38-win_amd64'),
+        PythonConfiguration(version='2.7.17', arch="32", identifier='cp27-win32', url=None),
+        PythonConfiguration(version='2.7.17', arch="64", identifier='cp27-win_amd64', url=None),
+        PythonConfiguration(version='3.5.4', arch="32", identifier='cp35-win32', url=None),
+        PythonConfiguration(version='3.5.4', arch="64", identifier='cp35-win_amd64', url=None),
+        PythonConfiguration(version='3.6.8', arch="32", identifier='cp36-win32', url=None),
+        PythonConfiguration(version='3.6.8', arch="64", identifier='cp36-win_amd64', url=None),
+        PythonConfiguration(version='3.7.5', arch="32", identifier='cp37-win32', url=None),
+        PythonConfiguration(version='3.7.5', arch="64", identifier='cp37-win_amd64', url=None),
+        PythonConfiguration(version='3.8.0', arch="32", identifier='cp38-win32', url=None),
+        PythonConfiguration(version='3.8.0', arch="64", identifier='cp38-win_amd64', url=None),
+        PythonConfiguration(version='2.7-v7.2.0', arch="32", identifier='pp272-win32', url='https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.2.0-win32.zip'),
+        PythonConfiguration(version='3.6-v7.2.0', arch="32", identifier='pp372-win32', url='https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.2.0-win32.zip'),
     ]
 
     if IS_RUNNING_ON_TRAVIS:
         # cannot install VCForPython27.msi which is needed for compiling C software
         # try with (and similar): msiexec /i VCForPython27.msi ALLUSERS=1 ACCEPT=YES /passive
-        python_configurations = [c for c in python_configurations if not c.version.startswith('2.7.')]
+        python_configurations = [c for c in python_configurations if not c.version.startswith('2.7')]
```

Comment:
If `PyPy` doesn't use the old `msvc` compiler as `CPython` does then, the build should  work. Was it failing at some point ?

Reply:
Apparently it does need that old compiler. I also hadn't thought of that yet, but at some point I got an error: https://travis-ci.org/joerick/cibuildwheel/jobs/608683633#L693

However, I'm now realizing: maybe that's a `setuptools`/`distutils` problem, and PyPy wouldn't actually need that?

Reply:
> However, I'm now realizing: maybe that's a setuptools/distutils problem, and PyPy wouldn't actually need that?

The download page of PyPy links to MSVC 2015 redistributable for both pp2/pp3 so I guess PyPy doesn't need that and it might be considered a `setuptools`/`distutils` problem.

Reply:
Good point! I'll look into it!

Reply:
@mattip How is PyPy 2.7 built, or where can I find the details/scripts of how releases are crafted?

The downloads section of PyPy 2.7 7.2.0 indeed refers to "Microsoft Visual C++ 2015 Redistributable Update 3 RC", but then https://pypy.readthedocs.io/en/latest/windows.html#installing-visual-compiler-v9-for-python-2-7 does refer to the "Microsoft Visual C++ Compiler for Python 2.7".

Reply:
PyPy 2.7 is still built with the old v9 compiler. We may reconsider this going forward, 

I don't know why we reccomend the 2015 redistributable for pypy2 as well, but apparently it works or someone would have complained

Reply:
Thanks! So @mayeut, we should keep skipping it on Travis CI, then, if we can't get that old MSVC installed there.